src/Microsoft.Data.SqlClient/tests/FunctionalTests/LocalizationTest.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data;
+using System.Data.Common;
+using System.Globalization;
+using System.Reflection;
+using System.Security;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.TDS.Servers;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.Tests
+{
+    public class LocalizationTest
+    {
+        [Fact]
+        public void Localization_EN_Test()
+        {
+            var localized = "";
+            var expected = "A network-related or instance-specific error occurred while establishing a connection to SQL Server.";
+
+            using TestTdsServer server = TestTdsServer.StartTestServer();
```

Comment:
Can you use the `GetLocalizedErrorMessage()` method call?

Reply:
LocalizationTest.cs has been refactored to use [Threory] and [InLineData].